src/main/java/com/github/checkstyle/regression/git/GitChange.java

```diff
@@ -19,6 +19,8 @@
 
 package com.github.checkstyle.regression.git;
 
+import org.eclipse.jgit.diff.DiffEntry;
```

Comment:
> So your suggestion is move GitChange to data package?

Yes, but I don't expect us to do this in a test only issue like I also stated before.

-------

For your other questions, they shouldn't be asked/answered in a test only PR/issue.
We will lose this discussion later on.
I will expand on this in a new post below.